config/initializers/wicked_pdf.rb

```diff
@@ -0,0 +1,4 @@
+WickedPdf.config = {
+  exe_path: ENV['wkhtmltopdf_binary_path'],
+  use_xvfb: true
```

Comment:
We shouldn't assume the use of xvfb. It'll break on systems that aren't using it (like our Docker image).

I think we should make this opt-in via another env var. Something like:

```ruby
WickedPdf.config = {
  exe_path: ENV['wkhtmltopdf_binary_path'],
  use_xvfb: ENV['wkhtmltopdf_use_xvfb'] == "true"
}
```

Reply:
Done - good catch, thanks!